pkgs/tools/networking/spiped/default.nix

```diff
@@ -2,33 +2,30 @@
 
 stdenv.mkDerivation rec {
   pname = "spiped";
-  version = "1.5.0";
+  version = "1.6.1";
 
   src = fetchurl {
     url    = "https://www.tarsnap.com/spiped/${pname}-${version}.tgz";
-    sha256 = "1mxcbxifr3bnj6ga8lz88y4bhff016i6kjdzwbb3gzb2zcs4pxxj";
+    sha256 = "8d7089979db79a531a0ecc507b113ac6f2cf5f19305571eff1d3413e0ab33713";
   };
 
   buildInputs = [ openssl ];
 
-  patchPhase = ''
+  postPatch = ''
     substituteInPlace libcperciva/cpusupport/Build/cpusupport.sh \
+      --replace "dirname" "${coreutils}/bin/dirname" \
       --replace "2>/dev/null" "2>stderr.log"
 
-    substituteInPlace POSIX/posix-l.sh       \
+    substituteInPlace libcperciva/POSIX/posix-l.sh       \
       --replace "rm" "${coreutils}/bin/rm"   \
-      --replace ">/dev/stderr" ">stderr.log" \
       --replace "2>/dev/null" "2>stderr.log"
 
-    substituteInPlace POSIX/posix-cflags.sh  \
-      --replace "rm" "${coreutils}/bin/rm"   \
-      --replace ">/dev/stderr" ">stderr.log" \
-      --replace "2>/dev/null" "2>stderr.log"
-  '';
```

Comment:
I think you didn't mean to remove the closing quotes and semicolon here.

Reply:
Also, after you fix this it would be great if you could squash the commits into 1 commit.

Reply:
You are correct, Sorry.  I have it working locally, but I don't have a way to push from my nixos machine into Github land, so I'm editing the file directly in the Github UI, which is less than ideal, and leads to these typos like this.  I guess I should have copy/pasted the entire file, instead of trying to surgically edit.

It appears to squash commits, requires me to be able to push from a local git clone, which I'm unable to do.  Unless you know of a way to do it in the GitHub Web UI?

Anyways, this latest commit puts back the '';